textworld/generator/chaining.py

```diff
@@ -508,7 +509,10 @@ def get_chains(state: State, options: ChainingOptions) -> Iterable[Chain]:
                 stack.append(child)
 
             if node.depth >= options.min_depth and node.breadth >= options.min_breadth:
-                yield chainer.make_chain(node)
+                chain = chainer.make_chain(node)
+
+                if len(chain.actions) >= options.chain_length:
```

Comment:
Is this supposed to be a min or a max?  Also you should pull the `make_chain()` in too so you avoid wasting the work if you're not going to yield it.

Reply:
I don't know actually. As a user what do you expect to get when running
`tw-make custom --quest-length 10 --quest-breadth 3` ?

Reply:
I figure we should have `min_length` and `max_length`, and `--quest-length` should set both

Reply:
I added those.